plugins/metrics/pkg/agent/plugin.go

```diff
@@ -43,6 +43,10 @@ func NewPlugin(ctx context.Context) *Plugin {
 		return drivers.NewExternalPromOperatorDriver(lg)
 	})
 
+	drivers.RegisterNodeDriverBuilder("otel-node-driver", func() (drivers.MetricsNodeDriver, error) {
```

Comment:
suggest renaming to something like "opni-manager-otel", what do you think?

Reply:
no objection :+1: